src/main/java/executor/command/CommandHelp.java

```diff
@@ -4,29 +4,49 @@
 import ui.Ui;
 import ui.Wallet;
 
+import java.security.spec.ECField;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class CommandHelp extends Command {
 
     /**
      * Constructor to provide the user with the details about the commands available.
      */
     public CommandHelp(String userInput) {
         this.userInput = userInput;
-        this.description = "Provides the user with all the available commands and descriptions.";
+        this.description = "Provides the user with all the available commands and descriptions.\n"
+                + "FORMAT : help \n";
         this.commandType = CommandType.HELP;
     }
 
     @Override
     public void execute(TaskList taskList) {
-        for (String s : CommandType.getNames()) {
-            if (!s.equals("ERROR") && !s.equals("TASK") && !s.equals("BLANK")) {
-                CommandType commandType = CommandType.valueOf(s);
-                Command c = Executor.createCommand(commandType, "null");
-                String commandDesc = c.getDescription();
+          String[] checkerForExtraInput;
+          checkerForExtraInput = userInput.split(" ");
+         try {
+             if (!checkerForExtraInput[1].isEmpty()) {
+                 System.out.println("Extra variable is added. Please follow -> FORMAT : help");
+                 return;
+             }
+         } catch (Exception e) {
```

Comment:
Looks like the catch part is empty. I believe this will give you a warning. In that case you can actually, remove the try catch block here as you are already checking for a particular condition.